datalake/src/main/java/com/sequenceiq/datalake/service/sdx/database/GcpDatabaseServerParameterSetter.java

```diff
@@ -26,6 +26,9 @@ public void setParameters(DatabaseServerV4StackRequest request, SdxDatabaseAvail
         } else {
             throw new IllegalArgumentException(availabilityType + " database availability type is not supported on Azure.");
         }
+        if (null != databaseEngineVersion && !databaseEngineVersion.isEmpty()) {
```

Comment:
Nit: Perhaps this could be simplified with `StringUtils.isNotEmpty(databaseEngineVersion)` like in `AwsDatabaseServerParameterSetter`.